### DISCUSSION TITLE
Which character has the "What's up, Doc?" catchphrase?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#988] [Galaxy Trivia] [answered]
**Bonus:**  Where in Hollywood can you find his "sign"?

**Comment by peterbaccaro** [comment]
The character **Bugs Bunny** has the catchphrase "What's up, Doc?".

Bugs Bunny has a star on the Hollywood Walk of Fame. Bugs Bunny's star is located at **7007 Hollywood Boulevard**, near the intersection of Hollywood Boulevard and Sycamore Avenue.

**Comment by thuongtruong1009** [comment, marked as answer]
The character with the catchphrase "What's up, Doc?" is Bugs Bunny. Bugs Bunny is a well-known animated character created by Warner Bros. He is an anthropomorphic rabbit known for his wit, sarcasm, and the ability to outsmart his adversaries. Bugs Bunny's catchphrase has become one of his signature lines over the years.

As for his "sign," Bugs Bunny's star on the Hollywood Walk of Fame can be found in Hollywood, California. The Hollywood Walk of Fame is a famous sidewalk embedded with stars honoring various individuals from the entertainment industry. Bugs Bunny received his star on the Hollywood Walk of Fame in 1985, recognizing his significant contributions to the world of animation and popular culture.
